src/src/rx_main.cpp

```diff
@@ -396,13 +402,14 @@ void ICACHE_RAM_ATTR ProcessRFPacket()
     {
     case RC_DATA_PACKET: //Standard RC Data Packet
         #if defined SEQ_SWITCHES
-        UnpackChannelDataSeqSwitches(&Radio, &crsf);
+        UnpackChannelDataSeqSwitches(Radio.RXdataBuffer, &crsf);
         #elif defined HYBRID_SWITCHES_8
-        UnpackChannelDataHybridSwitches8(&Radio, &crsf);
+        UnpackChannelDataHybridSwitches8(Radio.RXdataBuffer, &crsf);
         #else
         UnpackChannelData_11bit();
         #endif
-        crsf.sendRCFrameToFC();
+        if (connectionState == connected)
```

Comment:
Can we add some curly braces to the if statement? This will catch someone out for sure being embedded in a switch as well...